packages/org.json_schema.contrib/ref.pkl

```diff
@@ -27,8 +27,8 @@ local function parseJsonPointer(ref: String): List<String|Int> =
   ref
     .split("/")
     .map((str) -> str
-      .replaceAll("~0", "~")
-      .replaceAll("~1", "/"))
+    .replaceAll("~0", "~")
+    .replaceAll("~1", "/"))
```

Comment:
Revert; this method chain is within the `map()` so the indent is intentional

Reply:
This comment generally applies everywhere else, too. Do you mind undoing the formatting changes? I'm guessing some of these are committed by our auto-formatter, but IMO it makes it a little harder to read (we'll address all of this when we have a formatting tool that ships with Pkl)

Reply:
The keyboard shortcut is second nature, sorry 😅 I'll revert the lines I'm not otherwise touching.